types/fulu/data_column_sidecar.yaml

```diff
@@ -0,0 +1,34 @@
+Fulu:
+  DataColumnSidecar:
+    type: object
+    description: "A data column sidecar as defined in the Fulu CL spec."
+    required:
+      [
+        column_index,
```

Comment:
This is just `index` in the consensus specs: https://github.com/ethereum/consensus-specs/blob/ff99bc03d6da29d9ef6e055bdb8500e1b2942f1e/specs/fulu/das-core.md?plain=1#L81